doc/matter/conf.py

```diff
@@ -42,12 +42,12 @@
 html_last_updated_fmt = "%b %d, %Y"
 html_show_sourcelink = True
 html_show_sphinx = False
-html_title = "Matter documentation (nRF Connect SDK)"
+html_title = "Matter SDK documentation (nRF Connect platform)"
 
 html_theme_options = {
     "docset": "matter",
     "docsets": utils.ALL_DOCSETS,
-    "subtitle": "nRF Connect SDK",
+    "subtitle": "nRF Connect platform",
```

Comment:
I don't think this is the path to follow, let's discuss this first.

Reply:
Discussed today: subtitle was designed to make it clear that some doc sets come from upstream, but that what the user is browsing is actually part of nRF Connect SDK. Introducing a new concept, "platform", may confuse users.

Reply:
Right, fair point. I've reverted this particular line.